src/main/daos/ChallengeSetting.kt

```diff
@@ -85,8 +85,8 @@ data class ChallengeSettingNamespace(
     val description: String,
     val imageUrl: String,
     val sponsorName: String,
-    val expiration: DateTime,
-    val shareExpiration: DateTime,
+    val expiration: String,
+    val shareExpiration: String,
     val admin: Int,
```

Comment:
id try changing these to strings to see if primitives are the issue for klaxton

Reply:
which values? the rest of them that are not Strings?